examples/bind-shopping.amp.html

```diff
@@ -0,0 +1,454 @@
+            on="tap"
+            variable="selected"
+            value="{{8}}">
+        </amp-bind-event>
+        <amp-bind-attribute layout=nodisplay
+            attr="selected"
+            toggle="{{selected === 8}}">
+        </amp-bind-attribute>
+        <amp-bind-attribute layout=nodisplay
+            attr="out-of-stock"
+            toggle="{{!items[8].sizes.length}}">
+        </amp-bind-attribute>
+      </amp-img>
+    </div>
+
+    <amp-accordion>
+      <!--
+      <section expanded bind-attr="{expanded: !size}">
+      -->
+      <section expanded>
+        <amp-bind-attribute width=355 height=50
+            attr="expanded"
+            toggle="{{!size}}">
+
+          <!-- Nested here due to accordion's child limit. -->
+          <!--
+          <amp-img width=355 height=50 src="https://placehold.it/355x50?text=SIZE%20???"
+              bind-src="https://placehold.it/355x50?text=SIZE%20{{size}}">
+          -->
+          <amp-img width=355 height=50 src="https://placehold.it/355x50?text=SIZE%20???">
+            <amp-bind-attribute layout=nodisplay
+                attr="src"
+                value="{{'https://placehold.it/355x50?text=SIZE%20' + size}}">
+            </amp-bind-attribute>
+          </amp-img>
+
+        </amp-bind-attribute>
+
+        <div class="sizes">
+          <!--
+          <button
+              on="tap:setState({size: 'SMALL'})"
+              bind-class="{hidden: items[selected].sizes.indexOf('S') < 0}">
+          -->
+          <button>
+            <amp-bind-event layout=nodisplay
+                on="tap"
+                variable="size"
+                value="{{'SMALL'}}">
+            </amp-bind-event>
+            <amp-bind-attribute layout=nodisplay
+                attr="hidden"
+                toggle="{{items[selected].sizes.indexOf('S') < 0}}">
+            </amp-bind-attribute>
+            SMALL - 38"
+          </button>
+          <button>
+            <amp-bind-event layout=nodisplay
+                on="tap"
+                variable="size"
+                value="{{'MEDIUM'}}">
+            </amp-bind-event>
+            <amp-bind-attribute layout=nodisplay
+                attr="hidden"
+                toggle="{{items[selected].sizes.indexOf('M') < 0}}">
+            </amp-bind-attribute>
+            MEDIUM - 38"
+          </button>
+          <button>
+            <amp-bind-event layout=nodisplay
+                on="tap"
+                variable="size"
+                value="{{'LARGE'}}">
+            </amp-bind-event>
+            <amp-bind-attribute layout=nodisplay
+                attr="hidden"
+                toggle="{{items[selected].sizes.indexOf('L') < 0}}">
+            </amp-bind-attribute>
+            LARGE - 46"
+          </button>
+        </div>
+      </section>
+    </amp-accordion>
+
+    <!--
+    <button class="change-size"
+        on="tap:setState({size: ''})"
+        bind-class="{hidden: !size}">
+    -->
+    <button class="change-size">
```

Comment:
Selecting a size un-expands the `amp-accordion`'s section, and this button is displayed to allow modification. This is modeled off of one of Eric's use cases (Google Store's product pages for phones). Please check out the [video demo](https://drive.google.com/open?id=0Bxatm4FLOzANaEx2Y0NxVWFsV0k).